content/blog/2018-12-11-rcites.md

```diff
@@ -0,0 +1,83 @@
+
+Now, almost one year after, it appears to us that there were several key issues that the hackathon turned to be a success. The first was the short but thorough introduction to the development of R packages by Maëlle Salmon (Maëlle became an editor at rOpenSci shortly after). Within an hour or so we had a clear vision of what the development of an R package pipeline looks like. A second key was the help of more experienced people. Our group immensely benefited from the experience of [Tamora James](https://twitter.com/soayipm) that kindly answered our questions. The last one was the diversity of people interested in the project. Indeed, while some of us were beginner R users, others had moderate experience in creating packages and others had already dealt with CITES before, so we were able to split tasks organically. Some of us learned how to access the CITES/Speciesplus API (https://api.speciesplus.net/) and how to write a request via URLs. Others worked on rough versions of a functional code, yet again others started writing the help files and tested the package. After a few hours of frustration, ignoring why a given function refused to work as intended, things started to fit like pieces of a jigsaw puzzle. This was very empowering.   
+
+To summarize the hackathon: We learn a lot. First, how to create an R package, i.e. creating a structure of files, writing functions, and their documentation, testing functions, checking whether the package is working (it was already a lot at that stage of development). Second, how to make R writing URL requests to query a web API. Third, how to use GitHub for collaborative coding. Of course, we did not become experts on all of these topics, it was just the beginning of an adventure and a lot of work remained, but still … what a day!   
```

Comment:
Please clarify "Second, how to make R writing URL requests to query a web API. "

Do you mean "Second, how to write URL requests in R to query a web API."?